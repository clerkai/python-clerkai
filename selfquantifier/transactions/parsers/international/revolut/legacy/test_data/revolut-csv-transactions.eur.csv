Completed Date;Reference;Paid Out (EUR);Paid In (EUR);Exchange Out;Exchange In; Balance (EUR);Exchange Rate;Category
22. Jan 2021;Premium plan fee;7,99 ;;;;-4,19 ; ;General
22. Dec 2020;Premium plan fee;7,99 ;;;;85,34 ; ;General
26. Nov 2020;Wolt;44,40 ;;;;539,66 ; ;Restaurants
25. Nov 2020;MAX;16,52 ;;SEK 169,00 ;;584,06 ;FX-rate 1 € = 10,2346 SEK;Restaurants
22. Nov 2020;Premium plan fee;7,99 ;;;;617,43 ; ;General
14. Oct 2020;chrome;4,25 ;;USD 5,00 ;;840,11 ;FX-rate 1 € = 1,1792 $;Entertainment
6. Oct 2020;Bought EUR from SEK;;23 802,27 ;;SEK 250 000,00 ;23 822,28 ;FX-rate 1 € = 10,5032 SEK;General
22. Sep 2020;Sold EUR to BTC;2,00 ;;BTC 0,00022153 ;;0,01 ;FX-rate 1 € = 0,00011077 BTC ;General
21. Sep 2020;Premium plan fee;7,99 ;;;;2,01 ; ;General
21. Sep 2020;Money added via Apple Pay;;10,00 ;;;10,00 ; ;General
