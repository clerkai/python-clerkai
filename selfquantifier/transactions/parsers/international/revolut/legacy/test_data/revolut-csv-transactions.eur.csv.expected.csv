Real Date,Bank Date,Payee,Bank Message,Amount,Currency,Balance,Foreign Currency,Foreign Currency Amount,Foreign Currency Rate,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Currency,Raw Balance,Raw Foreign Currency,Raw Foreign Currency Amount,Raw Foreign Currency Rate
,2021-01-22,,Premium plan fee,-7.99,EUR,-4.19,,,,"{'Completed Date': '22. Jan 2021', 'Reference': 'Premium plan fee', 'Paid Out (EUR)': '7,99\xa0', 'Paid In (EUR)': nan, 'Exchange Out': nan, 'Exchange In': nan, ' Balance (EUR)': '-4,19\xa0', 'Exchange Rate': '\xa0', 'Category': 'General'}",,22. Jan 2021,,Premium plan fee,"-7,99 ",EUR,"-4,19 ",,,
,2020-12-22,,Premium plan fee,-7.99,EUR,85.34,,,,"{'Completed Date': '22. Dec 2020', 'Reference': 'Premium plan fee', 'Paid Out (EUR)': '7,99\xa0', 'Paid In (EUR)': nan, 'Exchange Out': nan, 'Exchange In': nan, ' Balance (EUR)': '85,34\xa0', 'Exchange Rate': '\xa0', 'Category': 'General'}",,22. Dec 2020,,Premium plan fee,"-7,99 ",EUR,"85,34 ",,,
,2020-11-26,,Wolt,-44.40,EUR,539.66,,,,"{'Completed Date': '26. Nov 2020', 'Reference': 'Wolt', 'Paid Out (EUR)': '44,40\xa0', 'Paid In (EUR)': nan, 'Exchange Out': nan, 'Exchange In': nan, ' Balance (EUR)': '539,66\xa0', 'Exchange Rate': '\xa0', 'Category': 'Restaurants'}",,26. Nov 2020,,Wolt,"-44,40 ",EUR,"539,66 ",,,
,2020-11-25,,MAX,-16.52,EUR,584.06,SEK,169.00,10.23460000000000,"{'Completed Date': '25. Nov 2020', 'Reference': 'MAX', 'Paid Out (EUR)': '16,52\xa0', 'Paid In (EUR)': nan, 'Exchange Out': 'SEK 169,00\xa0', 'Exchange In': nan, ' Balance (EUR)': '584,06\xa0', 'Exchange Rate': 'FX-rate 1\xa0€\u2008=\u200810,2346\xa0SEK', 'Category': 'Restaurants'}",,25. Nov 2020,,MAX,"-16,52 ",EUR,"584,06 ",SEK,"169,00","10,2346"
,2020-11-22,,Premium plan fee,-7.99,EUR,617.43,,,,"{'Completed Date': '22. Nov 2020', 'Reference': 'Premium plan fee', 'Paid Out (EUR)': '7,99\xa0', 'Paid In (EUR)': nan, 'Exchange Out': nan, 'Exchange In': nan, ' Balance (EUR)': '617,43\xa0', 'Exchange Rate': '\xa0', 'Category': 'General'}",,22. Nov 2020,,Premium plan fee,"-7,99 ",EUR,"617,43 ",,,
,2020-10-14,,chrome,-4.25,EUR,840.11,USD,5.00,1.17920000000000,"{'Completed Date': '14. Oct 2020', 'Reference': 'chrome', 'Paid Out (EUR)': '4,25\xa0', 'Paid In (EUR)': nan, 'Exchange Out': 'USD 5,00\xa0', 'Exchange In': nan, ' Balance (EUR)': '840,11\xa0', 'Exchange Rate': 'FX-rate 1\xa0€\u2008=\u20081,1792\xa0$', 'Category': 'Entertainment'}",,14. Oct 2020,,chrome,"-4,25 ",EUR,"840,11 ",USD,"5,00","1,1792"
,2020-10-06,,Bought EUR from SEK,23802.27,EUR,23822.28,SEK,250000.00,10.50320000000000,"{'Completed Date': '6. Oct 2020', 'Reference': 'Bought EUR from SEK', 'Paid Out (EUR)': nan, 'Paid In (EUR)': '23\xa0802,27\xa0', 'Exchange Out': nan, 'Exchange In': 'SEK 250\xa0000,00\xa0', ' Balance (EUR)': '23\xa0822,28\xa0', 'Exchange Rate': 'FX-rate 1\xa0€\u2008=\u200810,5032\xa0SEK', 'Category': 'General'}",,6. Oct 2020,,Bought EUR from SEK,"23 802,27 ",EUR,"23 822,28 ",SEK,"250 000,00","10,5032"
,2020-09-22,,Sold EUR to BTC,-2.00,EUR,0.01,BTC,0.00,0.00011077000000,"{'Completed Date': '22. Sep 2020', 'Reference': 'Sold EUR to BTC', 'Paid Out (EUR)': '2,00\xa0', 'Paid In (EUR)': nan, 'Exchange Out': 'BTC 0,00022153\xa0', 'Exchange In': nan, ' Balance (EUR)': '0,01\xa0', 'Exchange Rate': 'FX-rate 1\xa0€\u2008=\u20080,00011077\xa0BTC ', 'Category': 'General'}",,22. Sep 2020,,Sold EUR to BTC,"-2,00 ",EUR,"0,01 ",BTC,"0,00022153","0,00011077"
,2020-09-21,,Premium plan fee,-7.99,EUR,2.01,,,,"{'Completed Date': '21. Sep 2020', 'Reference': 'Premium plan fee', 'Paid Out (EUR)': '7,99\xa0', 'Paid In (EUR)': nan, 'Exchange Out': nan, 'Exchange In': nan, ' Balance (EUR)': '2,01\xa0', 'Exchange Rate': '\xa0', 'Category': 'General'}",,21. Sep 2020,,Premium plan fee,"-7,99 ",EUR,"2,01 ",,,
,2020-09-21,,Money added via Apple Pay,10.00,EUR,10.00,,,,"{'Completed Date': '21. Sep 2020', 'Reference': 'Money added via Apple Pay', 'Paid Out (EUR)': nan, 'Paid In (EUR)': '10,00\xa0', 'Exchange Out': nan, 'Exchange In': nan, ' Balance (EUR)': '10,00\xa0', 'Exchange Rate': '\xa0', 'Category': 'General'}",,21. Sep 2020,,Money added via Apple Pay,"10,00 ",EUR,"10,00 ",,,
